pkg/apm/applications_integration_test.go

```diff
@@ -3,114 +3,44 @@
 package apm
 
 import (
-	"os"
 	"testing"
 
-	"github.com/newrelic/newrelic-client-go/pkg/config"
+	"github.com/stretchr/testify/assert"
```

Comment:
Not a blocker, but we noticed that using `github.com/stretchr/testify/require` seems slightly better suited for integration tests. It has the same API as `assert`, but in situations where the assertion fails, `require` will stop execution and short circuit the test, which ensures the next step of the integration test won't happen (bc it would likely cause another error). Helps limit the noise with error output when running integration tests.

Reply:
Easy change.